docs/resources/Channel.md

```diff
@@ -579,6 +614,24 @@ once per 100 milliseconds, but will downsample so that no more than 256 datapoin
 As of 2023-04-14, clients upload a 1 channel, 48000 Hz, 32kbps Opus stream in an OGG container.
 The encoding, and the waveform details, are an implementation detail and may change without warning or documentation.
 
+
+### Message Snapshot Object
+
+#### Message Snapshot Structure
+
+| Field     | Type      | Description                                                                      |
+|-----------|-----------|----------------------------------------------------------------------------------|
+| message\* | [message] | subset of fields in the [message object](#DOCS_RESOURCES_CHANNEL/message-object) |
+
+\* The current list of message fields subset consists of:
+'type`, `content`, `embeds`, `attachments`, `timestamp`, `edited_timestamp`, `flags`, `mentions`, `mention_roles`.
```

Comment:
```suggestion
`type`, `content`, `embeds`, `attachments`, `timestamp`, `edited_timestamp`, `flags`, `mentions`, `mention_roles`.
```

Reply:
Is it intended that `mention_roles` is only returned on the Get Message(s) endpoints?